daal4py/sklearn/cluster/_k_means_0_23.py

```diff
@@ -213,7 +213,7 @@ def is_string(s, target_str):
         maxIterations=numIterations,
         accuracyThreshold=abs_tol,
         fptype=X_fptype,
-        resultsToEvaluate="computeCentroids",
+        resultsToEvaluate="computeCentroids|computeAssignments|computeExactObjectiveFunction",
```

Comment:
Is this what we want as default?

Reply:
yes, since daal4py always uses results from objective function, compute assignment is a prerequisite for objective function

Reply:
yes, and sounds good